server/http.go

```diff
@@ -54,26 +65,10 @@ func (p *Plugin) connectUserToZoom(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	channelID := r.URL.Query().Get("channelID")
-	if channelID == "" {
-		http.Error(w, "channelID missing", http.StatusBadRequest)
-		return
-	}
+	state := fmt.Sprintf("%s_%s", zoomStateKeyPrefix, userID)
```

Comment:
The state is a way to secure that the person that receives the token is the person that started the flow. That is the reason for every state to be unique (and that is why we were using a `model.NewId` to create the state). If the state is easily guessable, it could become a security threat.

Reply:
@larkox Good point, I added the random prefix back to the state: https://github.com/mattermost/mattermost-plugin-zoom/pull/150/commits/e606a33098d9e68ad25898138466c5a678f193a1

Please see https://github.com/mattermost/mattermost-plugin-zoom/pull/150#discussion_r463253784 and https://github.com/mattermost/mattermost-plugin-zoom/pull/150#discussion_r463256601 for more context.